draft-birkholz-cose-cometre-ccf-profile.md

```diff
@@ -98,30 +121,37 @@ where:
 - : denotes concatenation of lists
 - D\[k1:k2\] = D'_(k2-k1) denotes the list \{d'\[0\] = d\[k1\], d'\[1\] = d\[k1+1\], ..., d'\[k2-k1-1\] = d\[k2-1\]\} of length (k2 - k1).
 
-## Leaf Components
+## Transaction Components
 
 Each leaf in a CCF ledger carries the following components:
 
 ~~~
-CCF-leaf = [
-  internal-hash: bstr ; a string of HASH_SIZE bytes;
-  internal-data: bstr; a string of at most 1024 bytes; and
-  data_hash: bstr ; the serialization of the element stored at this leaf.
+ccf-leaf = [
+  internal-transaction-hash: bstr ; a string of HASH_SIZE bytes
+  internal-evidence: tstr         ; a string of at most 1024 bytes
+  data-hash: bstr                 ; a string of HASH_SIZE bytes
 ]
 ~~~
 
-The `internal_hash` and `internal_data` byte strings are internal to the CCF implementation. Similarly, the auxiliary tree entries are internal to CCF. They are opaque to receipt Verifiers, but they commit the TS to the whole tree contents and may be used for additional, CCF-specific auditing.
+The `internal-transaction-hash` and `internal-evidence` byte strings are internal to the CCF implementation. They can be safely ignored by receipt Verifiers, but they commit the TS to the whole tree contents and may be used for additional, CCF-specific auditing.
+
+`internal-transaction-hash` is a hash over the complete entry in the {{CCF-Ledger-Format}}, and `internal-evidence` is a revealable {{CCF-Commit-Evidence}} value that allows early persistence of ledger entries before distributed consensus can be established.
+
+`data-hash` summarises the subject of the proof: the data which is included in the ledger at this transaction.
 
 # CCF Inclusion Proofs
 
 CCF inclusion proofs consist of a list of digests tagged with a single left-or-right bit.
 
 ~~~
-CCF-inclusion-proof: [+ proof-element],
-
-proof-element = [
-  left: bool
-  hash: bstr
+ccf-inclusion-proof = bstr .cbor {
```

Comment:
To be validated and corrected as necessary

Reply:
![image](https://github.com/user-attachments/assets/e6032ce8-fa08-401e-b1d1-feaf8e23d724)